sonic_platform_base/sonic_storage/ssd.py

```diff
@@ -375,7 +375,10 @@ def parse_transcend_info(self):
                 self.temperature = temp_raw.split()[-1]
 
     def fetch_vendor_ssd_info(self, diskdev, model):
-        self.vendor_ssd_info = self._execute_shell(self.vendor_ssd_utility[model]["utility"].format(diskdev))
+        try:
+            self.vendor_ssd_info = self._execute_shell(self.vendor_ssd_utility[model]["utility"].format(diskdev))
+        except:
+            self.log.log_info("Error fetching vendor info. Falling back to generic info.")
```

Comment:
@assrinivasan can you handle this in the caller?

Reply:
Done